huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go

```diff
@@ -626,7 +636,7 @@ func resourceClusterRead(_ context.Context, d *schema.ResourceData, meta interfa
 		d.Set("highway_subnet_id", n.Spec.HostNetwork.HighwaySubnet),
 		d.Set("container_network_type", n.Spec.ContainerNetwork.Mode),
 		d.Set("container_network_cidr", flattenContainerNetworkCidrs(n.Spec.ContainerNetwork)),
-		d.Set("eni_subnet_id", n.Spec.EniNetwork.SubnetId),
+		d.Set("eni_subnet_id", flattenEniSubnetID(n.Spec.EniNetwork)),
 		d.Set("eni_subnet_cidr", n.Spec.EniNetwork.Cidr),
```

Comment:
what's the value of eni_subnet_id when we specify multi eni_subnet_id ?

Reply:
the description of eni_subnet_cidr is updated